src/utils/staleIssuesUtil.js

```diff
@@ -75,11 +74,35 @@ function getStaleIssues(org, repo) {
     })
 }
 
-
+function getRepoStaleIssues(org, repo) {
+    let  issueArray = [];
+    return getStaleIssues(org, repo)
+    .then((issues) => {
+        if(issues!=undefined && issues!=null) {
+            return issues;
+        }
+    }).then((issues)=>{
+            issues.map(function mappingToIssues(issue, index) {
+                let str = issue.repository_url;
+                let n = str.search(repo);
+                if(n!=-1 & n!=undefined) {
+                    if(!issueArray.includes(issue)){
+                        issueArray.push(issue);
+                    }
+                }
+            })
+            model_utils.setItem(repo,issueArray);
+            return issueArray;        
+    })
+    .catch((err) => {
+        throw err;
+    })
+}
 
 
 
 module.exports = {
     getOrgWideIssues: getOrgWideIssues,
-    getStaleIssues: getStaleIssues
+    getStaleIssues: getStaleIssues,
+    getRepoStaleIssues: getRepoStaleIssues
```

Comment:
This is great. Do we have to hook up this new function to the UI?

Reply:
Yes, we have to hook this up to the UI, We can just make changes to the community-toolbox.js and in place of calling getStaleIssues function we can call getRepoStaleIssues, which will return the stale issues corresponding to repo selected in drop down.